pkg/cmd/releasenotes/github.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2021 ScyllaDB
+
+package releasenotes
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	githubql "github.com/shurcooL/githubv4"
+)
+
+// PullRequest holds graphql data about a PR, including its commits and their contexts.
+type PullRequest struct {
+	Number githubql.Int
+	Author struct {
+		Login githubql.String
+	}
+	Title  githubql.String
+	Labels struct {
+		Nodes []struct {
+			Name githubql.String
+		}
+	} `graphql:"labels(first: 100)"`
+	Repository struct {
+		Name          githubql.String
+		NameWithOwner githubql.String
+		Owner         struct {
+			Login githubql.String
+		}
+	}
+	MergedAt githubql.DateTime
+}
+
+type PRNode struct {
+	PullRequest PullRequest `graphql:"... on PullRequest"`
+}
+
+type searchQuery struct {
+	RateLimit struct {
+		Cost      githubql.Int
+		Remaining githubql.Int
+	}
+	Search struct {
+		PageInfo struct {
+			HasNextPage githubql.Boolean
+			EndCursor   githubql.String
+		}
+		Nodes []PRNode
+	} `graphql:"search(type: ISSUE, first: 100, after: $searchCursor, query: $query)"`
+}
+
+const searchTimeFormat = "2006-01-02T15:04:05Z"
```

Comment:
There're couple of [constants](https://golang.org/pkg/time/#pkg-constants) used there to describe format, this one already uses 24h format (`3` or `03` is for 12h, `15` for 24h). Although I changed it to RFC3339 because it is also accepted by GH.

Reply:
+1 for RFC3339